extensions/sql-migration/package.json

```diff
@@ -2,15 +2,15 @@
   "name": "sql-migration",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "1.4.5",
+  "version": "1.4.6",
   "publisher": "Microsoft",
   "preview": false,
   "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
   "icon": "images/extension.png",
   "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
   "engines": {
     "vscode": "*",
-    "azdata": ">=1.41.0"
+    "azdata": ">=1.44.1"
```

Comment:
just curious: does this change specifically require an azdata update? or this is just to keep up with the latest version?

Reply:
this will require ads 1.44.1 to run.  I don't expect the extension to be visible until ads is >= 1.44.1